karmada-Failover-example/step4/text.md

```diff
@@ -0,0 +1,7 @@
+### Check the status and quantity distribution of pods and deployments
+
+RUN `karmadactl --kubeconfig /etc/karmada/karmada-apiserver.config get deployment `{{exec}}
```

Comment:
I think directly using `karmadactl get pods xxxx --watch` is a better idea, users could see the migration.

Reply:
you're right